collections/_testimonials/25-GOVIKING.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Numerical simulation of a cross-flow through a bundle to prepare the FSI simulations in the GO-VIKING European project"
+author: "Enrico Pivato"
+author_link: "mailto:enrico.pivato@vki.ac.be"
+organisation: "Von Karman Institute for Fluid Dynamics, Belgium"
+organisation_link: "https://www.vki.ac.be/"
+img: testimonial-25-GOVIKING.png
+---
+The GO-VIKING project was created to study flow-induced vibrations inside nuclear power plants more effectively. The VKI hosts a facility called GOKSTAD, where measurements are taken to compare with numerical results. The numerical results were obtained using LES simulations, with the aim of getting as close as possible to the experimental data and identifying correlations between the two.
```

Comment:
I would link to the project (and bring the "european" here, since it is not anymore in the title), and I would expand the VKI:


```suggestion
The [GO-VIKING European project](https://go-viking.eu/) was created to study flow-induced vibrations inside nuclear power plants more effectively. The Von Karman Institute hosts a facility called GOKSTAD, where measurements are taken to compare with numerical results. The numerical results were obtained using LES simulations, with the aim of getting as close as possible to the experimental data and identifying correlations between the two.
```